lib/modules/manager/composer/__fixtures__/composer6.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "acme/path-sources",
+  "description": "Fetch Packages via path",
+  "repositories": {
+    "awesome/path1": {
+      "type": "path",
+      "url": "awesome-path1"
+    }
```

Comment:
Can you give a more realistic test example? e.g. valid url at least

Reply:
Url in this context is a path on the file system relative to the dir of the composer.json file, see https://getcomposer.org/doc/05-repositories.md#path. This would be a valid "url" then, but I renamed things a bit so that the example makes more sense.